index.html

```diff
@@ -326,18 +326,48 @@ <h2>
         (at the user agent's discretion; see [[[#installability-signals]]]).
         </li>
       </ol>
+      <div class="note" data-link-for="WebAppManifest">
+        <p>
+          The {{Document}} might not be <a data-lt=
+          "within-scope-manifest">within scope</a> of the linked manifest, by
+          design. For example, a page at <code>https://example.com</code> might
+          link a manifest with <a>scope</a> <code>"/installable/"</code>,
+          allowing the top-level page of the site (which itself is not part of
+          the installable web application) to show installation UI.
+        </p>
+        <p>
+          Implementors need to be aware of this possibility. One consequence is
+          that if, post install, the user agent opens the document URL in an
+          <a>application context</a>, it would immediately start showing the
+          "out of scope" warning UI (that is recommended in
+          [[[#navigation-scope]]]). Hence, implementors might wish to avoid
```

Comment:
```suggestion
          <a>application context</a>, it would start exhibiting the
          "out of scope" behavior (that is recommended in
          [[[#navigation-scope]]]). Hence, implementors might wish to avoid
```